pkg/webui/components/form/field/field.styl

```diff
@@ -42,21 +42,20 @@
   .label
     font-weight: 600
 
-  &.horizontal
-    .title
-      padding-top: $cs.xxs
+    ::after
```

Comment:
```suggestion
    &:after
```

Reply:
Unfortunately, we cannot define custom rules and `stylint` does not have support for pseudo elements 🤷‍♂️

Reply:
 I don't quite get it. Why is this better? It works the way I added it.

Reply:
1. It is consistent with the rest of the code where we reference parent node via `&`
2. You have to use `&` to reference the parent selector
3. This works because this line is not needed at all

With `&`:
<img width="426" alt="Screenshot 2020-07-02 at 12 16 11" src="https://user-images.githubusercontent.com/16374166/86340635-e4831e00-bc5d-11ea-9123-519656329d99.png">

Without `&`:
<img width="426" alt="Screenshot 2020-07-02 at 12 16 26" src="https://user-images.githubusercontent.com/16374166/86340649-e9e06880-bc5d-11ea-889d-98871f09ee94.png">